### ISSUE TITLE
Even i have more productivity, what's wrong with you guys?

### ISSUE THREAD

**Issue by Fedcomp** [hlxbans/web#1]
Nothing to add.

**Comment by yamikaitou** [comment]
The project was scrapped...

**Comment by Fedcomp** [comment]
i know that, but it's just an amxx+web script, it's not a big deal to make it. Maybe 2-3 months of easy work (with frameworks ofc).

**Comment by yamikaitou** [comment]
Then make it yourself. We aren't working on it anymore

**Comment by Fedcomp** [comment]
Don't worry, on it. Good luck you guys anyway.

**Comment by yamikaitou** [comment]
Didn't notice your earlier comment completely before, but the project also included a SM plugin and an AMXX plugin with 1 web script to maintain both SM and AMXX actions

**Comment by Fedcomp** [comment]
yes i know. For first release it's better to be oriented just on amxx with knowledge of future expanding, or i'll end up the same way.

**Comment by yamikaitou** [comment]
The problem is that you have to develop the web interface to support both systems from the get-go if you want it to work correctly

**Comment by peternavr** [comment]
Woah, I didn't think this was still active.

**Comment by Fedcomp** [comment]
yes i know, but as far as i know it, i can code it the way where second system will be easyer to add than just when you rewrite it from amxx. So i know i'll support SM in the future, and i add additional code to current version for easyer coding in the future. It's mostly Admin system (they are different in Amxx and SM), nothing to change in Ban table.

But i'm surprised you don't have anything in repository.

@ClanStyles no, this project seems abandoned.

**Comment by peternavr** [comment]
Well I mean the github project is still ... active. Yeah we literally didn't write one line of code. The entire time who ever was handling the web interface kept saying "I have something I'll show you later" and never showed anyone. I think that's what caused it to go poof.

**Comment by Fedcomp** [comment]
I'm curious did you ever heard about GM Bans Amxbans edition (like pirated and improved version)? for some reason i think you never heard about it.

**Comment by peternavr** [comment]
Yeah I did. xPaw told me about it. He also had found an exploit in it... and they weren't happy about it.

**Comment by yamikaitou** [comment]
GM Bans sucks, it isn't improved at all.
